rosa_backing_up_and_restoring_applications/backing-up-applications.adoc

```diff
@@ -15,28 +15,39 @@ This is a two stage process:
 
 include::modules/oadp-preparing-aws-credentials.adoc[leveloffset=+1]
 
-include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
+include::modules/installing-oadp-rosa-sts.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 [id="oadp-addtl-resources_{context}"]
 .Additional resources
 
 * xref:../rosa_backing_up_and_restoring_applications/backing-up-applications.adoc#oadp-preparing-aws-credentials_rosa-backing-up-applications[Preparing AWS credentials]
 
+//Added the following two modules to make a more complete guide.
+
+[id="oadp-rosa-backing-up-and-cleaning-example"]
+== Backing up workloads on OADP with ROSA STS
+
+include::modules/performing-a-backup-oadp-rosa-sts.adoc[leveloffset=+2]
+
+include::modules/cleanup-a-backup-oadp-rosa-sts.adoc[leveloffset=+2]
+
 [id="rosa-backing-up-applications-known-issues"]
 == Known issues
-.Restic, Kopia, and DataMover are not supported or recommended
 
-* link:https://issues.redhat.com/browse/OADP-1054[CloudStorage: openshift-adp-controller-manager crashloop seg fault with Restic enabled]
-* (Affects OADP 1.1.x_ only): link:https://issues.redhat.com/browse/OADP-1055[CloudStorage: bucket is removed on CS CR delete, although it doesn't have "oadp.openshift.io/cloudstorage-delete": "true"]
+* Restic, Kopia, and DataMover are not supported or recommended.
+* link:https://issues.redhat.com/browse/OADP-1054[CloudStorage: openshift-adp-controller-manager crashloop seg fault with Restic enabled].
```

Comment:
Consider referencing the Jira number as the link meta data. Examples: https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-known-issues_release-notes

Reply:
This is not one of my topics. I'm not sure how this got pulled into this PR? :face_with_diagonal_mouth: